crates/red_knot_python_semantic/src/types.rs

```diff
@@ -1132,6 +1135,9 @@ pub enum KnownClass {
     GenericAlias,
     ModuleType,
     FunctionType,
+    // Typing
+    Sized,
+    Hashable,
```

Comment:
Sorry to catch this only after you'd done the work, but I don't see any reason for these to be known classes? We should only use `KnownClass` when either a) the type has special-cased behavior that is not reflected in its typeshed definition, or b) we will commonly need to conjure this type "from thin air" within the type inference code itself, usually because the type is builtin and sometimes created "from thin air" by the runtime (e.g. `bool`) or because it is closely related to other special-cased types (e.g. `KnownClass::Int` and `Type::IntLiteral`, etc.)

I don't think `Sized` or `Hashable` meet either of those criteria.

If you want to use these types (or any other typeshed type) in tests, you don't need it to be a `KnownClass`; that's just a minor convenience. You can just use `typing_symbol_ty` (added in this PR, and something we'll definitely want) directly in the test.

Reply:
I hadn't quite internalized that I only actually really need `typing_symbol_ty`, thanks for pointing that out. Will remove  the`KnownClass` entries